lib/src/cpp/array/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+add_library(tick_array EXCLUDE_FROM_ALL
```

Comment:
Can we define at the top of this file:
set(ARRAY_INCLUDE_DIR, "${PROJECT_SOURCE_DIR}/lib/include/tick/array/")
?

Reply:
we can - you're the CMake guy! ^_^

Reply:
Done
https://github.com/Dekken/tick/blob/re-org/lib/src/cpp/array/CMakeLists.txt

Reply:
I've done it for other CMakeLists files as well